.eslintrc.yaml

```diff
@@ -86,6 +87,15 @@ rules:
   dot-notation: [0]
   eol-last: [2]
   eqeqeq: [2]
+  eslint-comments/disable-enable-pair: [2]
+  eslint-comments/no-aggregating-enable: [2]
+  eslint-comments/no-duplicate-disable: [2]
+  eslint-comments/no-restricted-disable: [0]
+  eslint-comments/no-unlimited-disable: [2]
+  eslint-comments/no-unused-disable: [2]
+  eslint-comments/no-unused-enable: [2]
+  eslint-comments/no-use: [0]
+  eslint-comments/require-description: [0]
```

Comment:
I'm actually all for enabling this rule.
If PR authors need to disable eslint, they should justify why.
But I understand that it needs quite a bit of work to enable, so can be postponed for now.

Reply:
We have [this](https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/no-unlimited-disable.html) which will require specific rule name to disable, which I think is sufficient.

Comment syntax inside a eslint disable comment is also pretty weird and I didn't know about it until very recently, so it may confuse people on how to even put the requested comment.